test/test.lua

```diff
@@ -3688,6 +3688,55 @@ function test.catArray()
    end
 end
 
+-- designed to specifically hit the batched kernel for catArray
```

Comment:
maybe have a test where the output is non-contiguous to begin with (since you support that I think?)